docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,269 @@
 
-5. If the flowsheet is later modified, click the Refresh Graph button to
-   see the changes.
+.. _ifv-screenshot:
 
-The displayed layout is preserved as much as possible, with new components
-appearing along a diagonal line. 
+.. only:: html
 
-.. note::
-    This feature is still under development. 
-    Several types of changes to the flowsheet currently cause the entire user-
-    modified layout to be lost. Consider saving the layout often (see below).
+    **In the screenshot of the IFV UI below, you can click on one of the indicated areas to jump to the associated
+    description.**
 
-.. image:: ../../_images/modelvis/new_unit_model_layout.png
+    .. raw:: html
 
-6. Save the displayed layout using the save button on the visualization page. 
-   This writes the visualization to a file in the user's home directory under 
-   `.idaes/viz` using the model name provided to `visualize()`. 
-   In this example the filename would be `model_name.viz`.
+        <img src="../../_images/ifv_screenshot_window.png" usemap="#image-map" width="800px">
+        <map name="image-map">
+            <area target="_self" alt="Top bar" title="Top bar" href="#top-bar" coords="20,10,800,100" shape="rect"></area>
+            <area target="_self" alt="Diagram Controls" title="Diagram Controls" href="#diagram-controls" coords="484,110,800,165" shape="rect"></area>
+            <area target="_self" alt="Diagram/Flowsheet" title="Diagram/Flowsheet" href="#diagram" coords="25,175,800,570" shape="rect"></area>
+            <area target="_self" alt="Stream Table" title="Stream Table" href="#stream-table" coords="25,570,800,890" shape="rect"></area>
+        </map>
 
-.. _streamlabels:
+.. only:: latex
 
-Stream Labels
--------------
+    .. figure:: ../../_images/ifv_screenshot_window.png
+        :width: 800
 
-The initial layout loads with the stream labels hidden. Show or hide all of 
-the stream labels by clicking the button with the speech bubbles, 
-on the toolbar.
+        Screenshot of the main window of the IFV UI
 
-Show or hide an individual label by right clicking on the stream or its label.
+Top bar
++++++++
 
-.. _miscfeatures:
+.. figure:: ../../_images/ifv_screenshot_topbar.png
+    :width: 600
 
-Misc. Features
---------------
+    Screenshot of the top bar of the IFV UI
 
-* Right click on an icon to rotate it by 90 degrees.
+The *top bar* has a title bar, which contains the IDAES logo and the name of the flowsheet being visualized,
+and a menu. The menu items are:
 
-* Create anchor points on a stream by left clicking on the stream. The stream 
-  will be forced to connect through each anchor point, typically adding right angles.
+.. _export-menu:
 
+* **Refresh**: Update the view with any changes made to the flowsheet from the Python side.
+  This also has the effect of saving the current layout.
+* **Save**: Save the current layout to the data store that was specified with the visualization
```

Comment:
What happens if the user doesn't specify a file (`save_as=name.json`) invoking the `visualize` method? Does this actually save anything?